sdk.d.ts

```diff
@@ -2,7 +2,7 @@
  * @file
  * Types for Simple Icons SDK.
  */
-// eslint-disable-next-line n/file-extension-in-import
```

Comment:
This removal I think that happens because you've a different version of xo on your PATH, perhaps because you're using another version of Node.js from the one that installed deps locally. I've experienced it one time. Revert it and the lint will pass.

Reply:
Yes. Fixed now.